src/user_templates_api/templates/jupyter_lab/templates/blank_r/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Should we add a `job_types` field which is an array of the compatible job type ids from https://workspaces.api.hubmapconsortium.org/job_types/? We can specify them for the non-jupyter_lab templates and backfill after the fact?